### PR TITLE
update documentation in README.md

### PR DESCRIPTION
**Pull request by felixerdy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,72 @@
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/a3b0b564-6d90-4bbf-9b5f-2f5fd46d5a97/deploy-status)](https://app.netlify.com/sites/smart-city-dashboard/deploys)
 
+This project contains the source code of the frontend of the smart city dashboard. The display can connect to the [backend](https://github.com/reedu-reengineering-education/smart-city-dashboard-backend) and show different kinds of environmental and smart city related data. The dashboard contains three different pages: dashboard view, map view and an info page.
```
Comment (Smart-City-Muenster) [inline README.md:5]: shows

File: README.md [hunk 2]
```diff
@@ -2,8 +2,72 @@
...
+
+1. Clone the repo
+2. `cd smart-city-dashboard`
+3. `cp .env.example .env`
+4. Change the env variable `REACT_APP_API_URL` to the URL of the backend
+5. `yarn` or `npm install`
+6. `yarn build` or `npm build`
+7. Serve the `build` folder with your favorite webserver
+
+### Development
```
Comment (Smart-City-Muenster) [inline README.md:17]: Vorschlag: 
The same as running the project, instead of
6. `yarn start` or `npm start`

File: README.md [hunk 3]
```diff
@@ -2,8 +2,72 @@
...
+
+- Tile: Displaying numbers with units and titles in tiles / cards
+- Progress: Visualizing progress which is here used to show the utilization of parking decks
+- Line diagram: visualizing (multiple) lines in a diagram
+
+#### How to add a new data source
+
+If you want to add a new data source to the dashboard, there are multiple steps neccessary:
+
+1. Start in the `src/reducers` folder and create a new reducer similar to the existing ones. Here, you will also define the state for youry particular dataset
```
Comment (Smart-City-Muenster) [inline README.md:60]: youry -> your

File: README.md [hunk 4]
```diff
@@ -2,8 +2,72 @@
...
+2. Next one, create corresponding actions for your data in the `src/actions` folder
+3. After defining how the data should be stored and managed, we can take care of fething the data. Create a new saga in the `src/sagas` folder similar to the existing files. The generator functions will fetch data regularly based on the interval you can define
+
+#### Show the data on the dashboard
+
+1. The most laborious will take place in `src/views` where you create a new container for your data. Here, you need to access the redux state and render different kinds of visualizations
```
Comment (Smart-City-Muenster) [inline README.md:66]: labour

File: README.md [hunk 5]
```diff
@@ -2,8 +2,72 @@
...
+2. You can now import your container in `pages/Home` to display it on the main page. You might need to adjust the layout here
```
Comment (Smart-City-Muenster) [inline README.md:67]: ... the overall page layout here ...

**Review by Smart-City-Muenster — APPROVED** [review]
Danke! Meine Kommentare sind nur Kleinigkeiten. Sobald die repariert sind, gerne mergen!

File: README.md (hunk 1 above)
Comment (felixerdy) [reply README.md]: Resolved in [ca1fb3b](https://github.com/reedu-reengineering-education/smart-city-dashboard/pull/15/commits/ca1fb3b0457e9f148eef04857de7522da4ca8815)

File: README.md (hunk 2 above)
Comment (felixerdy) [reply README.md:17]: Resolved in [ca1fb3b](https://github.com/reedu-reengineering-education/smart-city-dashboard/pull/15/commits/ca1fb3b0457e9f148eef04857de7522da4ca8815)

File: README.md (hunk 3 above)
Comment (felixerdy) [reply README.md]: Resolved in [ca1fb3b](https://github.com/reedu-reengineering-education/smart-city-dashboard/pull/15/commits/ca1fb3b0457e9f148eef04857de7522da4ca8815)

File: README.md (hunk 4 above)
Comment (felixerdy) [reply README.md]: Resolved in [ca1fb3b](https://github.com/reedu-reengineering-education/smart-city-dashboard/pull/15/commits/ca1fb3b0457e9f148eef04857de7522da4ca8815)

File: README.md (hunk 5 above)
Comment (felixerdy) [reply README.md]: Resolved in [ca1fb3b](https://github.com/reedu-reengineering-education/smart-city-dashboard/pull/15/commits/ca1fb3b0457e9f148eef04857de7522da4ca8815)

**Pull request merged**